modules/st2-elastic-field/directive.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+angular.module('main')
+  .directive('st2ElasticField', [
```

Comment:
What are the pros of such a move? I see at least one benefit of the separation: it's future-proof. But I don't see any cons.